dt-impact-tracer/src/main/java/edu/washington/cs/dt/impact/data/TestData.java

```diff
@@ -0,0 +1,119 @@
+package edu.washington.cs.dt.impact.data;
+
+import edu.washington.cs.dt.RESULT;
+import edu.washington.cs.dt.TestExecResult;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class TestData {
+    private final String dependentTest;
+    private final RESULT intended;
+    private final Set<String> beforeTests;
+    private final Set<String> afterTests;
+    private final RESULT revealed;
+
+    private final List<String> revealingOrder;
+
+    public TestData(final String dependentTest,
+                    final RESULT intended,
+                    final Set<String> beforeTests,
+                    final Set<String> afterTests,
+                    final RESULT revealed,
+                    final List<String> revealingOrder) {
+        this.dependentTest = dependentTest;
+        this.intended = intended;
+        this.beforeTests = beforeTests;
+        this.afterTests = afterTests;
+        this.revealed = revealed;
+        this.revealingOrder = revealingOrder;
+    }
+
+    @Override
+    public String toString() {
+        // If the intended behavior is the revealed behavior, then the before/after tests are the
+        // tests needs before/after to make it have the intended behavior.
```

Comment:
Can you elaborate under what scenario would the intended behavior be the same as the revealed behavior?